decode/clarity.ts

```diff
@@ -227,10 +227,6 @@ export async function replay(
                 let boxModelEvent = entry as BoxModelEvent;
                 r.boxmodel(boxModelEvent.data, iframe);
                 break;
-            case Event.Target:
```

Comment:
Are you removing this? That would break things.

Reply:
No, this is only for rendering Target events in visualization of lean mode. This will have no impact to the data you receive, we are still parsing Event.Target like before.